porch/controllers/remoterootsync/pkg/controllers/remoterootsyncset/remoterootsync_controller.go

```diff
@@ -136,27 +162,94 @@ func (r *RemoteRootSyncSetReconciler) applyToClusterRef(ctx context.Context, sub
 }
 
 // BuildObjectsToApply config root sync
-func BuildObjectsToApply(subject *api.RemoteRootSyncSet) ([]*unstructured.Unstructured, error) {
+func (r *RemoteRootSyncSetReconciler) BuildObjectsToApply(ctx context.Context, subject *api.RemoteRootSyncSet) ([]*unstructured.Unstructured, error) {
+	// TODO: stop requiring this to be set & source from a deployment instead
+	gcpProjectID := os.Getenv("GCP_PROJECT_ID")
+	imageName := oci.ImageTagName{
+		Image: "us-west1-docker.pkg.dev/" + gcpProjectID + "/deployment/myfirstnginx",
```

Comment:
lgtm if the TODO on line 166 extends to line 169 as well. Otherwise hardcoding the image location is worth reconsidering (in a follow-up)

Reply:
Extended the TODO to make it clear that it does.  My goal is to get it from _a_ package into the cluster, then add the mechanisms that are actually more important to us, such as how this package is specified and how we update.